src/target/source/codegen_cuda.cc

```diff
@@ -297,19 +297,10 @@ std::string CodeGenCUDA::Finish() {
   decl_stream << "#define TVM_ENABLE_L2_PREFETCH 0\n";
   decl_stream << "#endif\n";
 
-  decl_stream << "\n#ifdef _WIN32\n";
```

Comment:
Why change this?

Reply:
It's because NVSHMEM contains `#include <cstdint>`, which is in conflict with the original `#define int64_t long long` and could lead to CUDA compilation error. The `#define` semantics is quite error prone, so I remove it and just do `using`.